src/js/webrequest.js

```diff
@@ -607,14 +607,28 @@ let getWidgetList = (function () {
         continue;
       }
       let replace = widget.domains.some(domain => {
-        if (!tabData.origins.hasOwnProperty(domain)) {
-          return false;
+        // checks for wildcard in domain
+        if (domain[0] == "*") {
+          // reduce wildcard domain from "*.domain" to "domain"
+          domain = domain.slice(2);
+          if (!tabData.origins.hasOwnProperty(domain)) {
```

Comment:
This doesn't address the problem described in https://github.com/EFForg/privacybadger/pull/2613/files#r435451266